README.Rmd

```diff
@@ -618,13 +618,20 @@ We're constantly making minor tweaks to improve the model's accuracy, speed, and
 
 # Usage
 
+There are two ways of running the model:
+
+* [On a local machine](#running-the-model-locally-all-users) (available to all users)
+* [In the cloud via AWS Batch](#running-the-model-on-aws-batch-ccao-staff-only) (only available to CCAO staff)
+
+## Running the model locally (all users)
```

Comment:
**mega nitpick:** The rest of the README uses title case for all headings, even sub headings. Let's keep it consistent for these too.